low_H: 0
high_H: 255
low_S:  0
high_S: 30
low_V: 225
high_V: 255
scale_factor_: 34.687